components/registry-facade/pkg/registry/metrics.go

```diff
@@ -36,6 +36,9 @@ func (m *measuringRegistryRoundTripper) RoundTrip(req *http.Request) (*http.Resp
 
 	if strings.Contains(req.URL.Path, "/manifests/") {
 		m.metrics.ManifestHist.Observe(dt.Seconds())
+		if err != nil {
+			m.metrics.ManifestFailedCounter.Inc()
+		}
 	} else if strings.Contains(req.URL.Path, "/blobs/") {
 		m.metrics.BlobCounter.Inc()
```

Comment:
can you add a label to the error metric and also increment here? (`blobs`,`manifests`)